locmap/docs/admin.md

```diff
@@ -0,0 +1,61 @@
+#Admin Resources
+
+##Description
+Administrative API calls
+
+***
+##Requires Admin Authentication
+* Requires Admin Authentication to access any of the below methods
+
+***
+
+##`GET` /admin/:userId/crashReport/:osType/:year/:month
+Get crash reports for given OS and time period
+
+###Parameters
+- **userId** _(required)_ - Id of current user (must be admin)
+- **OS type** _(required)_ - Get crash reports from this OS
+	* Choices - 'android', 'ios', 'wp'
+- **year** _(required)_ - Get crash reports from this year
+- **month** _(required)_ - Get crash reports from this month
+
+###Return
+- Returns status and result of action
+
+***
+
+##`POST` /admin/:userId/accountRecovery
+Puts account into recovery mode
+
+###Parameters
+- **userId** _(required)_ - Id of current user (must be admin)
+- **request body** _(required)_ - Included in this body is:
+	#####body
```

Comment:
I'm not sure what the hashes are supposed to be, but it doesn't seem to render correctly at least in GitHub.